app/assets/stylesheets/common.scss

```diff
@@ -1078,4 +1083,12 @@ div.secondary-actions {
   }
 }
 
+.auth-container {
+  max-width: 660px;
+}
+
+.nav-z-100 {
+  z-index: 100;
+}
```

Comment:
Is it necessary? It's used on a tab list inside the container with the globe image, but the image is background. There's nothing else that might cover the tabs.

Reply:
Not strictly necessary, but it depends on the length of text.

If the labels "Signup" and "Login" are long texts, the globe's arm would come behind the text, making the text of unselected tab somewhat obfuscated by the image behind it here is an example:

![image](https://github.com/openstreetmap/openstreetmap-website/assets/99971018/3c5d99e2-61ff-4dd4-b2d0-48c6569a9e62).

Reply:
Doesn't that happen with `.nav-z-100` too?

Reply:
Actually, it looks like z-index does not make any difference here. Thanks!